src/e2e/resources/simple-temple-expected/simple-temple-test-group/dao/dao.go

```diff
@@ -77,7 +77,7 @@ func executeQuery(db *sql.DB, query string, args ...interface{}) (int64, error)
 
 // CreateSimpleTempleTestGroup creates a new simpleTempleTestGroup in the datastore, returning the newly created simpleTempleTestGroup
 func (dao *DAO) CreateSimpleTempleTestGroup(input CreateSimpleTempleTestGroupInput) (*SimpleTempleTestGroup, error) {
-	row := executeQueryWithRowResponse(dao.DB, "INSERT INTO simple_temple_test_group () VALUES ();", input.ID, input.AuthID)
+	row := executeQueryWithRowResponse(dao.DB, "INSERT INTO simple_temple_test_group (id) VALUES ($1) RETURNING id;", input.ID, input.AuthID)
```

Comment:
I see here there's also `input.AuthID` passed in (i.e. a `$2`) - is that going to be used?

Reply:
That's my problem to fix, project not having any auth isn't handled by DAO generation - also made a card for a few days ago 😉